completions/openssl

```diff
@@ -46,7 +46,8 @@ _comp_cmd_openssl()
 
     if ((cword == 1)); then
         local commands
-        commands="$("$1" help 2>&1 | command sed -e '/commands\|help:/d')"
+        # We only want the standard commands, so we delete everything starting after and including "Message Digest commands"
+        commands="$("$1" help 2>&1 | command sed -E -e '/Standard commands|help:/d' -e '/Message Digest commands/,$d')"
```

Comment:
Maybe this is fine, but `-E` was added to POSIX recently. GNU/BSD `sed`'s are fine, but it might cause an issue with some other environments. However, `\|` in the original BRE is not POSIX. We could split the regular expression as `/Standard commands/d;/help:/d`

Reply:
Changed to your suggestion